frontend/src/pages/EventDashboard/Hackerpack/index.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import {
+  Box,
+  Avatar,
+  Typography,
+  Button,
+  Grid,
+  Divider
+} from "@material-ui/core";
+
+import Markdown from "components/generic/Markdown";
+
+import PageHeader from "components/generic/PageHeader";
+import PageWrapper from "components/layouts/PageWrapper";
+
+import { OutboundLink } from "react-ga";
+
+import styles from "../EventDashboard.module.scss";
+
+import Partners from "constants/hackerpack-partners.js";
+
+const Hackerpack = () => {
+  const CompanySection = ({ name, icon, description, link }) => {
```

Comment:
<CompanySection /> should be defined outside of <HackerPack />, otherwise it's recreated on every render. So let's just move it into it into its own file and then also define the styles with material-ui useStyles-pattern, I was supposed to get rid of EventDashboard.module.scss anyway ;)